omnibus/files/private-chef-cookbooks/private-chef/spec/libraries/helper_spec.rb

```diff
@@ -114,6 +114,64 @@
     end
 
     context 'when elastic search is unavailable' do
+      it 'should return 2 but the request fails first time only' do
```

Comment:
That would also work, though lately I have a preference for shared context in the same file vs dynamically created tests, for ease of understandability and to keep the amount of test-related "logic" to a minimum.